script/fix_permissions.sh

```diff
@@ -15,4 +15,6 @@ else
   set -e
   # Change ownership of all files inside the specific folder from root/root to current user/group
   docker run -v "${LOCATION}":/beat ${DOCKER_IMAGE} sh -c "find /beat -user 0 -exec chown -h $(id -u):$(id -g) {} \;"
+  # Change permissions with write access of all files inside the specific folder
+  docker run -v "${LOCATION}":/beat ${DOCKER_IMAGE} sh -c "find /beat -exec chmod -R +w {} \;"
```

Comment:
This command is somehow too expensive, it added 4 minutes 

![image](https://user-images.githubusercontent.com/2871786/109143361-d8e9e780-7757-11eb-8705-a43d40001575.png)